source/fundamentals/crud/read-operations/cursor.txt

```diff
@@ -7,7 +7,7 @@ Access Data From a Cursor
 Read operations that return multiple documents do not immediately return
 all values matching the query. Because a query can potentially match
 very large sets of documents, these operations rely upon an
-object called a :node-api:`cursor <Cursor.html>`
+object called a :node-api-4.0:`cursor </classes/findcursor.html>`
```

Comment:
Should cursor be updated to `FindCursor` here?
Possibly something like "a cursor called ``FindCursor``"?

Applies to all instances

Reply:
Great question, I think cursor should be replaced with `FindCursor` in the find.txt page. I think this instance of cursor just should not link to the cursor API class, and instead should be a definition of the database concept of a [cursor](https://en.wikipedia.org/wiki/Cursor_(databases)#:~:text=In%20computer%20science%2C%20a%20database,and%20removal%20of%20database%20records.).

I think defining cursor succinctly is out of scope for this ticket, so I just removed the link.

Fixes:

Update Find: https://github.com/mongodb/docs-node/pull/175/commits/ba06d580c6daed6fd83ae469cf2f027d1df30299
Update Cursor Page:  https://github.com/mongodb/docs-node/pull/175/commits/83666adb734f74b7e26acea5df877584e6c831ae